src/render/processor.rs

```diff
@@ -61,7 +61,11 @@ impl<'a> AudioParamValues<'a> {
 
     /// Get the computed values for the given [`crate::param::AudioParam`]
     ///
-    /// For both A & K-rate params, it will provide a slice of length [`crate::RENDER_QUANTUM_SIZE`]
+    /// For A-Rate params, the slice will be of length [`crate::RENDER_QUANTUM_SIZE`]
+    /// For K-Rate params, the slice will be of length 1
+    ///
+    /// This is compliant with the AudioWorklet specification, cf.
+    /// <https://www.w3.org/TR/webaudio/#audioworkletprocess-callback-parameters>
     pub fn get(&self, index: &AudioParamId) -> &[f32] {
         &self.get_raw(index).channel_data(0)[..]
```

Comment:
We should actually return a slice of size 1 here for k-rate obviously. The rest of the buffer is garbage per
```

        param_computed_values
            .channel_data_mut(0)
            .copy_from_slice(param_intrisic_values);
```
Where `param_intrisic_values` is of size 1 only (via tick)

Reply:
You are right, didn't spot this one